test/e2e/framework/resourceinterpretercustomization.go

```diff
@@ -7,15 +7,28 @@ import (
 	"github.com/onsi/ginkgo/v2"
 	"github.com/onsi/gomega"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	kubeclient "k8s.io/client-go/kubernetes"
+	"k8s.io/klog/v2"
 
 	configv1alpha1 "github.com/karmada-io/karmada/pkg/apis/config/v1alpha1"
 	karmada "github.com/karmada-io/karmada/pkg/generated/clientset/versioned"
+	pkgutil "github.com/karmada-io/karmada/pkg/util"
 )
 
 // CreateResourceInterpreterCustomization creates ResourceInterpreterCustomization with karmada client.
-func CreateResourceInterpreterCustomization(client karmada.Interface, customization *configv1alpha1.ResourceInterpreterCustomization) {
+func CreateResourceInterpreterCustomization(client karmada.Interface, kubeClient kubeclient.Interface,
+	customization *configv1alpha1.ResourceInterpreterCustomization) {
 	ginkgo.By(fmt.Sprintf("Creating ResourceInterpreterCustomization(%s)", customization.Name), func() {
-		_, err := client.ConfigV1alpha1().ResourceInterpreterCustomizations().Create(context.TODO(), customization, metav1.CreateOptions{})
+		var err error
+		func() {
+			defer pkgutil.PrintTimeCost("CreateResourceInterpreterCustomizationE2ETest")()
+			_, err = client.ConfigV1alpha1().ResourceInterpreterCustomizations().Create(context.TODO(), customization, metav1.CreateOptions{})
+		}()
+
+		svc, errSvc := kubeClient.CoreV1().Services("karmada-system").Get(context.TODO(), "karmada-webhook", metav1.GetOptions{})
+		ep, errEp := kubeClient.CoreV1().Endpoints("karmada-system").Get(context.TODO(), "karmada-webhook", metav1.GetOptions{})
+		klog.Infof("karmada-webhook svc: %+v, errSvc: %+v\n\nep: %+v, errEp: %+v", svc, errSvc, ep, errEp)
```

Comment:
I don't understand why we need to output this information.

Reply:
+1

Reply:
Since the `karmada-webhook.karmada-system.svc:443` is called and failed, I wanna whether the status of this service or the related endpoints is healthy, although, it may has little significance.

I have ran this line in my forked repository, what amazed me is that the actual output of this line is `services "karmada-webhook" not found`、`endpoints "karmada-webhook" not found`. So impossible, the service must exist, while I can `get` it by `kubeClient`.

But, it is not important, so I won't dwell on it anymore.